eng/common/scripts/tag-reviewer.ps1

```diff
@@ -0,0 +1,24 @@
+param(
+    [Parameter(Mandatory = $true)]
+    $RepoOwner,
+
+    [Parameter(Mandatory = $true)]
+    $RepoName,
+
+    [Parameter(Mandatory = $true)]
+    $GitHubUser,
+
+    [Parameter(Mandatory = $true)]
+    $PRNumber,
+  
+    [Parameter(Mandatory = $true)]
+    $AuthToken,
+)
+
+$headers = @{
+  Authorization = "bearer $AuthToken"
+}
+
+
+# sample
+# https://api.github.com/repos/MicrosoftDocs/azure-docs-sdk-python/pulls/677/requested_reviewers
```

Comment:
This doesn't exist yet, I wanted to get eyes on the factoring of the dotnet piece early. Swapping over to the pshell side.